codalab/worker/worker_monitoring.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import os
+from typing import Dict, Optional
+
+import sentry_sdk
+from sentry_sdk.profiler import start_profiling
+
+from .worker_run_state import RunState
+
+logger = logging.getLogger(__name__)
+
+# Only do profiling in dev and test environments
+# And sample a lower percentage of transactions.
+transaction_sample_rate = float(os.getenv('CODALAB_SENTRY_TRANSACTION_RATE') or 0)
+profiles_sample_rate = float(os.getenv('CODALAB_SENTRY_PROFILES_RATE') or 0)
+assert 0 <= transaction_sample_rate <= 1
+assert 0 <= profiles_sample_rate <= 1
+sentry_sdk.init(
```

Comment:
Should we move this profiling init code to a separate file (sentry_init.py or something) so we can reuse it?